package_manifest.yaml

```diff
@@ -1111,3 +1111,67 @@ pulpcore_packages:
     python-whitenoise: {}
     python-yarl: {}
     python-zipp: {}
+
+foreman_release_packages:
+  vars:
+    package_base_dir: 'packages/foreman/'
+    repoclosure_config: repoclosure/yum.conf
+    repoclosure_target_repos:
```

Comment:
Does this need to target the main foreman repository as well?

Reply:
I think so, adding.